images/flask.svg

```diff
@@ -0,0 +1,151 @@
+           id="rect10-6"
+           width="226.25748"
+           height="83.886711"
+           x="657.93054"
+           y="180.05664"
+           ry="5.6997771" />
+        <flowRoot
+           xml:space="preserve"
+           id="flowRoot56"
+           style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:40px;line-height:125%;font-family:Akkurat-Bold;-inkscape-font-specification:'Akkurat-Bold Bold';letter-spacing:0px;word-spacing:0px;fill:#333333;fill-opacity:1;stroke:#333333;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+           transform="matrix(0.26458333,0,0,0.26458333,7.481251,161.9098)"><flowRegion
+             id="flowRegion58"
+             style="fill:#333333;stroke:#333333"><rect
+               id="rect60"
+               width="803.02771"
+               height="213.92097"
+               x="2483.5386"
+               y="200.70723"
+               style="fill:#333333;stroke:#333333" /></flowRegion><flowPara
+             id="flowPara62"
+             style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:66.66666412px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:center;writing-mode:lr-tb;text-anchor:middle;fill:#333333;stroke:#333333">web app</flowPara></flowRoot>      </g>
+      <g
+         id="g4958">
+        <flowRoot
+           xml:space="preserve"
+           id="flowRoot1000-5"
+           style="font-style:normal;font-variant:normal;font-weight:bold;font-stretch:normal;font-size:40px;line-height:125%;font-family:Akkurat-Bold;-inkscape-font-specification:'Akkurat-Bold Bold';letter-spacing:0px;word-spacing:0px;fill:#000000;fill-opacity:1;stroke:none;stroke-width:1px;stroke-linecap:butt;stroke-linejoin:miter;stroke-opacity:1"
+           transform="matrix(0.26458333,0,0,0.26458333,34.146994,176.78145)"><flowRegion
+             id="flowRegion1002-3"><rect
+               id="rect1004-73"
+               width="932.27899"
+               height="415.6915"
+               x="1267.2537"
+               y="102.80756" /></flowRegion><flowPara
+             id="flowPara1006-2"
+             style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333">two-way C++ to Python bindings</flowPara></flowRoot>        <flowRoot
```

Comment:
Could you remove `two way` we are just calling C++ from Python and we are not calling Python from C++.

```suggestion
             style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:53.33333588px;font-family:'Roboto Condensed';-inkscape-font-specification:'Roboto Condensed, Normal';font-variant-ligatures:normal;font-variant-caps:normal;font-variant-numeric:normal;font-feature-settings:normal;text-align:start;writing-mode:lr-tb;text-anchor:start;fill:#333333">C++ to Python bindings</flowPara></flowRoot>        <flowRoot
```

Same goes for other occurences

Reply:
'two ways' removed in next commit